libs/shared/connected/src/lib/SignatureLists/useGetSignatureLists.ts

```diff
@@ -25,10 +28,47 @@ export const GetCurrentCollection = gql`
     }
   }
 `
+export const GetOpenLists = gql`
+  query allOpenLists {
+    signatureCollectionAllLists {
```

Comment:
```suggestion
    signatureCollectionAllOpenLists {
```
Use this instead, this one is meant for admins.